package.json

```diff
@@ -19,9 +19,9 @@
     "gen-sql": "IASQL_ENV=ci ts-node ./node_modules/.bin/typeorm migration:generate -n",
     "migrate": "IASQL_ENV=ci ts-node ./node_modules/.bin/typeorm migration:run",
     "gen-module": "IASQL_ENV=ci ./.module_migration.sh",
-    "docker-build": "docker build -t iasql:latest .",
-    "docker-run": "docker run -e IASQL_ENV=local -p 5432:5432 --name iasql iasql:latest",
-    "docker-run-vol": "docker run -e IASQL_ENV=local -p 5432:5432 -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
+    "docker-build": "DOCKER_DEFAULT_PLATFORM=linux/amd64 docker build -t iasql:latest .",
+    "docker-run": "DOCKER_DEFAULT_PLATFORM=linux/amd64 docker run -e IASQL_ENV=local -p 5432:5432 --name iasql iasql:latest",
+    "docker-run-vol": "DOCKER_DEFAULT_PLATFORM=linux/amd64 docker run -e IASQL_ENV=local -p 5432:5432 -v $(dirname $PWD)/db:/var/lib/postgresql/14/main --name iasql iasql:latest",
```

Comment:
Are all the systems running on amd64? Like our servers. If not, this is going to cause the build/run to be slower.